bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/OpenWebNetBindingConstants.java

```diff
@@ -133,6 +133,10 @@
     public static final String CHANNEL_CU_BATTERY_STATUS = "batteryStatus";
     public static final String CHANNEL_CU_WEEKLY_PROGRAM_NUMBER = "weeklyProgram";
     public static final String CHANNEL_CU_SCENARIO_PROGRAM_NUMBER = "scenarioProgram";
+    public static final String CHANNEL_CU_FAILURE_DISCOVERED = "failureDiscovered";
+    public static final String CHANNEL_CU_AT_LEAST_ONE_PROBE_OFF = "atLeastOneProbeOFF";
```

Comment:
What about changing `atLeastOneProbeOFF ` -> `atLeastOneProbeOff ` similar to the other channels?
see other comment below